packages/runtime/src/helpers/functions.ts

```diff
@@ -71,10 +97,11 @@ export const generateFunctions = async (
       join(__dirname, '..', '..', 'lib', 'templates', 'handlerUtils.js'),
       join(functionsDir, functionName, 'handlerUtils.js'),
     )
+    writeGeneratorField(functionName, functionTitle, functionsDir)
   }
 
-  await writeHandler(HANDLER_FUNCTION_NAME, false)
-  await writeHandler(ODB_FUNCTION_NAME, true)
+  await writeHandler(HANDLER_FUNCTION_NAME, 'Next.js SSR handler', false)
+  await writeHandler(ODB_FUNCTION_NAME, 'Next.js ISR handler', true)
```

Comment:
[dust] Worth moving these into constants at the top of the module or in the constants file?